### DISCUSSION TITLE
不过如此 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#266] [Announcements]
# 不过如此 — 码志

当我健康的时候，我常会想，要是更有钱多好，可以选择不同的生活，想躺平可以躺平；当我不得不在病床上「躺平」的时候，我又感觉，健康真好，有健康做前提，那些选项才有意义。

https://mazhuang.org/2023/08/02/just-so-so/

**Comment by dzvision** [comment]
加油，身体健康。

**Reply by mzlogin** [reply]
谢谢～
